addons/opsportal/opsportal.yaml

```
@@ -6,10 +6,10 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: opsportal
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.0-10"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.0-12"
```

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.0-11"
```
skipped a revision? 🤔

---

Yeah I wasn't here, the [4.0 version](https://github.com/mesosphere/kubernetes-base-addons/pull/976) had it 12.
How should we handle this, @joejulian maybe you have some thoughts?

---

I _think_ 11 would be ok in this case since they are essentially the same. If we were bumping to a new version of the chart here I might have a different opinion. but also curious what @joejulian thinks.

---

Lets be consistent. The same revision matches between releases, let's use that to inform ourselves when we forget about this next week.